probe/print.go

```diff
@@ -28,9 +29,8 @@ func renderOutputInTableWithNoBorders(data [][]string) {
 // printDaemonsetData function
 func printDaemonsetData(daemonsetStatus *Status) {
 	var data [][]string
-
-	color.Green("\nFound KubeArmor running in Kubernetes\n\n")
-	_, err := boldWhite.Printf("Daemonset :\n")
+	fmt.Fprintf(os.Stdout, "\033[32m\nFound KubeArmor running in Kubernetes\033[0m\n\n")
+	_, err := printWhiteBoldText("Daemonset :\n")
 	if err != nil {
 		color.Red(" Error while printing")
```

Comment:
will this cause issue when error hits, since we are using `color` mod?

Reply:
No, it's just that the error will not be saved in file!

Reply:
I think it will come under the refactor! I will try to refactor all the `color` with a print function!